### PR TITLE
Adapting the release documentation for kmesh v0.2.0

### PR DESCRIPTION
**Pull request by lec-bit**
**What type of PR is this?**
documentation

**What this PR does / why we need it**:
Adapting the release documentation for kmesh v0.2.0
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: README-zh.md [hunk 1]
```diff
@@ -70,32 +69,42 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
...
-  # The arm image for Kmesh online compilation and execution, supports OS kernel versions 5.10 and above.
-  # arm镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
-  docker pull ghcr.io/kmesh-net/kmesh-arm:v0.1.0
+  docker pull ghcr.io/kmesh-net/kmesh:v0.2.0
+  
+  # 在线编译镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
+  docker pull ghcr.io/kmesh-net/kmesh-compatible:v0.2.0
+  
+  # kmesh编译环境镜像
+  docker pull ghcr.io/kmesh-net/kmesh-build-x86:v0.2.0
```
Comment (nlgwcy) [inline README-zh.md:85]: v0.2.0 -> leatest

File: docs/kmesh_compile-zh.md [hunk 2]
```diff
@@ -101,218 +16,59 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
...
+    Kmesh的镜像编译需要准备好kmesh源码，以及kmesh-build镜像，镜像可以通过如下命令获取
 
-    ```sh
-    [root@dev Kmesh]# ll build/docker/
-    total 12K
-    -rw-r--r--. 1 root root  793 Nov 25 01:31 kmesh.dockerfile
-    -rw-r--r--. 1 root root 1.5K Nov 25 10:48 kmesh.yaml
-    -rw-r--r--. 1 root root  764 Nov 25 01:31 start_kmesh.sh
+    ```bash
+    docker pull ghcr.io/kmesh-net/kmesh-build-x86:v0.2.0
```
Comment (nlgwcy) [inline docs/kmesh_compile-zh.md:22]: kmesh-build image version need match kmesh-code version

File: docs/kmesh_compile-zh.md [hunk 3]
```diff
@@ -101,218 +16,59 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
...
+  [root@dev tmp]# cd kmesh/
+  [root@dev Kmesh]# make build
   ```
 
-### docker image [兼容模式](../build/docker/README.md)构建
-
-- 准备工作
-
-  - docker-engine安装
+  kmesh会在编译镜像中进行编译构建，并将编译产物输出至out目录
```
Comment (nlgwcy) [inline docs/kmesh_compile-zh.md:40]: add example

File: docs/kmesh_compile-zh.md [hunk 4]
```diff
@@ -101,218 +16,59 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
...
-- 启动Kmesh服务
+  可以由用户指定参数构建，示例如下：
 
-  ```sh
-  [root@ ~]# systemctl start kmesh.service
-  # 查看Kmesh服务运行状态
-  [root@ ~]# systemctl status kmesh.service
+  ```bash
+  #用户自定义HUB TARGET TAG内容，若未指定则采用默认值
+  [root@localhost kmesh]# make docker HUB=ghcr.io/kmesh-net TARGET=kmesh TAG=v0.2.0
```
Comment (nlgwcy) [inline docs/kmesh_compile-zh.md:58]: normally, only need specify tag, hub/target had default value.

File: docs/kmesh_compile-zh.md [hunk 5]
```diff
@@ -101,218 +16,59 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
...
-  ```
-
-  查看本地镜像仓库已有Kmesh镜像
+- Kmesh编译清理
 
   ```sh
-  [root@dev docker]# docker images
-  REPOSITORY            TAG                 IMAGE ID            CREATED             SIZE
-  kmesh                 v0.1.0              fd60c89b1253        4 days ago          1.5GB
+  [root@dev Kmesh]# make clean
```
Comment (nlgwcy) [inline docs/kmesh_compile-zh.md:45]: `make clean` needs a separate section

File: README-zh.md (hunk 1 above)
Comment (hzxuzhonghu) [reply README-zh.md:85]: latest

File: README-zh.md [hunk 6]
```diff
@@ -70,32 +69,42 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
...
   # 用于openEuler 23.03 OS的Kmesh x86镜像
-  docker pull ghcr.io/kmesh-net/kmesh:v0.1.0
-
-  # x86镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
-  docker pull ghcr.io/kmesh-net/kmesh-x86:v0.1.0
-
-  # The arm image for Kmesh online compilation and execution, supports OS kernel versions 5.10 and above.
-  # arm镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
-  docker pull ghcr.io/kmesh-net/kmesh-arm:v0.1.0
+  docker pull ghcr.io/kmesh-net/kmesh:v0.2.0
```
Comment (hzxuzhonghu) [inline README-zh.md:79]: should we put it under L65

File: README-zh.md [hunk 7]
```diff
@@ -70,32 +69,42 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
...
+  
+  # 在线编译镜像用于Kmesh的在线编译以及执行，支持的OS版本为5.10及以上
+  docker pull ghcr.io/kmesh-net/kmesh-compatible:v0.2.0
```
Comment (hzxuzhonghu) [inline README-zh.md:82]: I think this should be under L73 - 针对所有OS版本：

File: README-zh.md (hunk 1 above)
Comment (hzxuzhonghu) [reply README-zh.md:85]: We donot need to pull explicitly, i think when we run make docker/make build, it will be pulled automatically

**Review by hzxuzhonghu — COMMENTED** [review]
/lgtm

File: README-zh.md [hunk 8]
```diff
@@ -57,45 +57,41 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
...
 
-  -  Helm安装方式
+  默认使用名为 ghcr.io/kmesh-net/kmesh:latest的镜像，如需使用兼容模式或其他版本可自行修改
 
+  -  Helm安装方式
+  
    ```sh
   [root@ ~]# helm install kmesh ./deploy/helm -n kmesh-system --create-namespace
-  ```
+   ```
```
Comment (hzxuzhonghu) [inline README-zh.md:89]: nit: remove the white space

File: README-zh.md [hunk 9]
```diff
@@ -57,45 +57,41 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
...
 
   - Yaml安装方式
-
+
```
Comment (hzxuzhonghu) [inline README-zh.md:92]: ditto

File: README.md [hunk 10]
```diff
@@ -107,20 +102,27 @@ Note: * Planning
...
+  time="2024-02-19T10:16:53Z" level=info msg="command StartServer successful" subsys=manager
+  time="2024-02-19T10:16:53Z" level=info msg="start write CNI config\n" subsys="cni installer"
+  time="2024-02-19T10:16:53Z" level=info msg="kmesh cni use chained\n" subsys="cni installer"
+  time="2024-02-19T10:16:54Z" level=info msg="Copied /usr/bin/kmesh-cni to /opt/cni/bin." subsys="cni installer"
+  time="2024-02-19T10:16:54Z" level=info msg="kubeconfig either does not exist or is out of date, writing a new one" subsys="cni installer"
+  time="2024-02-19T10:16:54Z" level=info msg="wrote kubeconfig file /etc/cni/net.d/kmesh-cni-kubeconfig" subsys="cni installer"
+  time="2024-02-19T10:16:54Z" level=info msg="command Start cni successful" subsys=manager
   ```
-
+
```
Comment (hzxuzhonghu) [inline README.md:125]: rm white space

File: docs/kmesh_compile.md [hunk 11]
```diff
@@ -6,209 +6,79 @@ The Kmesh needs to be compiled and built in the Linux environment with the Kmesh
...
 ## build
 
-### build from source
-
-- Code download
-
-  ```sh
-  [root@ ~]# git clone https://github.com/kmesh-net/kmesh.git
-  ```
+prerequisite
```
Comment (hzxuzhonghu) [inline docs/kmesh_compile.md:9]:
```suggestion
### prerequisite
```

File: docs/kmesh_compile.md [hunk 12]
```diff
@@ -6,209 +6,79 @@ The Kmesh needs to be compiled and built in the Linux environment with the Kmesh
...
 
-- Code compilation
+- install docker-engine
 
   ```sh
-  [root@ ~]# cd kmesh/
-  [root@ ~]# ./build.sh -b
+  [root@dev Kmesh]# yum install docker-engine
```
Comment (hzxuzhonghu) [inline docs/kmesh_compile.md:14]: nit: we may add a link to docker website

File: docs/kmesh_compile.md [hunk 13]
```diff
@@ -6,209 +6,79 @@ The Kmesh needs to be compiled and built in the Linux environment with the Kmesh
...
+  User-defined HUB, TARGET, TAG values. If not specified, default values will be used.
+  HUB=ghcr.io/kmesh-net
+  TARGET=kmesh
+  TAG= #git sha
+  
+  [root@localhost kmesh]# make docker
+  ...
+  Successfully tagged ghcr.io/kmesh-net/kmesh:b68790eb07830e757f4ce6d1c478d0046ee79730
+  
+  [root@localhost kmesh]# make docker HUB=ghcr.io/kmesh-net TARGET=kmesh TAG=v0.2.0
```
Comment (hzxuzhonghu) [inline docs/kmesh_compile.md:68]: The host name is not consistent

**Pull request merged**